src/instruments/view/instrumentspaneltreemodel.cpp

```diff
@@ -42,7 +42,16 @@ InstrumentsPanelTreeModel::InstrumentsPanelTreeModel(QObject* parent)
 {
     context()->currentMasterNotationChanged().onNotify(this, [this]() {
         IMasterNotationPtr masterNotation = context()->currentMasterNotation();
-        m_masterNotationParts = masterNotation ? masterNotation->parts() : nullptr;
+        if (masterNotation) {
```

Comment:
We can slightly simplify this condition:

m_masterNotationParts = nullptr;

if (masterNotation) {
m_masterNotationParts = masterNotation->parts();
m_masterNotationParts->partsChanged().onNotify(this, [this]() { ... }); 
}

It helps to remove the nested else block

Reply:
This is a relic of my past where every `if` should have an `else` :-).
But I will change it.

Reply:
Done